src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -814,6 +815,43 @@ namespace
             object_assembly_instance->transform_sequence()
                 .set_transform(0.0, transform);
 
+            if (node->GetMotBlurOnOff(time) && node->MotBlur() == 1)
+            {
+                bool is_animated = false;
+                Control* tm_controller;
+                Control* controller;
+                tm_controller = node->GetTMController();
+                auto class_id = tm_controller->ClassID();
+
+                controller = tm_controller->GetPositionController();
```

Comment:
Reusing ("recycling") variables is not ideal.

Much more compact and without variable recycling:

    const bool is_animated =
        tm_controller->GetPositionController()->IsAnimated() > 0 ||
        tm_controller->GetRotationController()->IsAnimated() > 0 ||
        tm_controller->GetScaleController()->IsAnimated() > 0;